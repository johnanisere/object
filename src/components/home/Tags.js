import React from "react"
import OptionLayout from "../SectionLayout"
import List from "../../blocks/FlexibleContainer"

const App = () => (
    <OptionLayout title="TAGS">
        <List list="true" />
    </OptionLayout>)

export default App