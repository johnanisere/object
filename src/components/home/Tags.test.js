
import React from "react";
import { shallow } from "enzyme";

import Tags from "./Tags"
import OptionLayout from "../SectionLayout"
import List from "../../blocks/FlexibleContainer"


const props = {
    displayData: [],
}

it("Tags component exists", () => {
    const wrapper = shallow(<Tags {...props} />);
    expect(wrapper.exists()).toBe(true);
});

it("Tags component renders", () => {
    const wrapper = shallow(<Tags {...props} />);
    expect(wrapper).toMatchSnapshot();
});

it(`Tags should contain "List", "Tags", "OptionLayout"`, () => {
    const wrapper = shallow(<Tags {...props} />);

    expect(wrapper.find(List)).toHaveLength(1);
    expect(wrapper.find(OptionLayout)).toHaveLength(1);
});