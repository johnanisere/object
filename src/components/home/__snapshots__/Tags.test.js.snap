// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tags component renders 1`] = `ShallowWrapper {}`;
